src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Components/ContextMenuHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Input;
+using Microsoft.CmdPal.Ext.WindowWalker.Commands;
+using Microsoft.CmdPal.Ext.WindowWalker.Helpers;
+using Microsoft.CmdPal.Ext.WindowWalker.Properties;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.WindowWalker.Components;
+
+internal sealed class ContextMenuHelper
+{
+    internal static List<CommandContextItem> GetContextMenuResults(in WindowWalkerListItem listItem)
+    {
+        if (!(listItem?.Window is Window windowData))
+        {
+            return new List<CommandContextItem>(0);
+        }
+
+        var contextMenu = new List<CommandContextItem>()
+        {
+            new(new CloseWindowCommand(windowData))
+            {
+                // AcceleratorKey = Key.F4,
```

Comment:
`RequestedShortcut(new(true, false, false, false, Vkey.F4, 0))` is what you're looking for 😉

Reply:
oh my goodness -- when did those come along. This is cool.